Code/Numerics/EigenSolvers/PowerEigenSolver.cpp

```diff
@@ -42,6 +37,11 @@ bool powerEigenSolver(unsigned int numEig, DoubleSymmMatrix &mat,
   bool converged = false;
   unsigned int i, j, id, iter, evalId;
 
+  constexpr unsigned int MAX_ITERATIONS = 1000;
+  constexpr double TOLERANCE = 0.001;
+  constexpr double HUGE_EIGVAL = 1.0e10;
+  constexpr double TINY_EIGVAL = 1.0e-10;
```

Comment:
```suggestion
  const unsigned int MAX_ITERATIONS = 1000;
  const double TOLERANCE = 0.001;
  const double HUGE_EIGVAL = 1.0e10;
  const double TINY_EIGVAL = 1.0e-10;
```
and please move these to the top of the function so that they are a bit more obvious